sp/src/game/client/clientshadowmgr.cpp

```diff
@@ -1468,7 +1468,8 @@ void CClientShadowMgr::InitDepthTextureShadows()
 		m_DummyColorTexture.InitRenderTargetSurface( r_flashlightdepthres.GetInt(), r_flashlightdepthres.GetInt(), IMAGE_FORMAT_BGR565, true );
 #else
 #if defined(MAPBASE) //&& !defined(ASW_PROJECTED_TEXTURES)
-		m_DummyColorTexture.InitRenderTarget( m_nDepthTextureResolution, m_nDepthTextureResolution, RT_SIZE_OFFSCREEN, nullFormat, MATERIAL_RT_DEPTH_NONE, false, "_rt_ShadowDummy" );
+		// SAUL: we want to create a render target of specific size, so use RT_SIZE_NO_CHANGE
+		m_DummyColorTexture.InitRenderTarget( m_nDepthTextureResolution, m_nDepthTextureResolution, RT_SIZE_NO_CHANGE, nullFormat, MATERIAL_RT_DEPTH_NONE, false, "_rt_ShadowDummy" );
```

Comment:
I recall something about this particular line change from the VDC fixes causing issues with Mapbase's projected textures. I'm not sure if I'm misremembering or thinking of a different line of code, but did you re-add Saul's fix here because you thought it was a mistake or because it's actually needed?

Reply:
Maybe you're thinking of `depthTex`, which is initialized a few lines down (line 1497 on this branch)? Because it has the same fix applied as here. Like the commit states, this change is taken from [Entropy : Zero](https://github.com/entropy-zero/source-sdk-2013/blob/138ebbe61a5979326a0d62c23b60ee9b8404f840/sp/src/game/client/clientshadowmgr.cpp#L1362).
When I tested my binaries (on Linux) the flashlight was cut off - only one corner was showing. Eventually I figured out that if `r_flashlightdepthres` was larger than my vertical screen resolution it would bug out - smaller or equal and it'd look fine.
I knew Entropy : Zero also had some projected texture enhancements and the bug was not present there, so I diffed the two files. After I changed this line it worked fine with larger resolutions.